app/controllers/requests_controller.rb

```diff
@@ -47,8 +46,4 @@ def set_requester_on(request)
     request.requester.name = current_user.name
     request.requester.email = current_user.email
   end
-
-  def persistence_to_zendesk_necessary?
```

Comment:
meh.

I'd prefer to make them use a separate code path entirely at that stage rather than do magic metaprogramming.